docs/make.jl

```
@@ -9,6 +14,8 @@ makedocs(
         "API" => "fft.md",
         "Examples" => "examples.md",
     ],
+    warnonly=[:missing_docs],
```

What's missing?

---

I think the docstring should be added to the documentation, since it's mentioned in the README of the package and the main page of the docs, rather than ignoring the error message.

---

I believe as well that all the docs should be included. But this was an orthogonal issue.
For this reason, I believe it essential to build the docs with a new version of documenter, which has the capability of catching this...

---

Add the reference to https://github.com/JuliaMath/FFTW.jl/blob/7d0f38c58eb861d79994ec632373eea4639f8a23/docs/src/index.md?plain=1#L20

---

Added